### PR TITLE
Add control event articles

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

- Added events overview for controls.
- Added How to handle events.
- Moved a common image to the _shared_ folder.

Contributes to #76

@luisquintanilla 

Preview Links: 

File Type | File Name | Published Url
-- | -- | --
Content | dotnet-desktop-guide/net/winforms/controls/events.md | https://review.docs.microsoft.com/en-us/dotnet/desktop/winforms/controls/events?view=netdesktop-5.0&branch=pr-en-us-1113
Content | dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md | https://review.docs.microsoft.com/en-us/dotnet/desktop/winforms/controls/how-to-add-an-event-handler?view=netdesktop-5.0&branch=pr-en-us-1113
Content | dotnet-desktop-guide/framework/winforms/controls/how-to-add-an-event-handler.md | https://review.docs.microsoft.com/en-us/dotnet/desktop/winforms/controls/how-to-add-an-event-handler?view=netframeworkdesktop-4.8&branch=pr-en-us-1113
Content | dotnet-desktop-guide/net/winforms/controls/how-to-add-a-picture-to-a-control.md | https://review.docs.microsoft.com/en-us/dotnet/desktop/winforms/controls/how-to-add-a-picture-to-a-control?view=netdesktop-5.0&branch=pr-en-us-1113
Content | dotnet-desktop-guide/net/winforms/controls/how-to-set-the-display-text.md | https://review.docs.microsoft.com/en-us/dotnet/desktop/winforms/controls/how-to-set-the-display-text?view=netdesktop-5.0&branch=pr-en-us-1113
Content | dotnet-desktop-guide/net/winforms/how-to-connect-multiple-events-to-a-single-event-handler-in-windows-forms.md | https://review.docs.microsoft.com/en-us/dotnet/desktop/winforms/how-to-connect-multiple-events-to-a-single-event-handler-in-windows-forms?view=netdesktop-5.0&branch=pr-en-us-1113
Content | dotnet-desktop-guide/net/winforms/how-to-create-event-handlers-at-run-time-for-windows-forms.md | https://review.docs.microsoft.com/en-us/dotnet/desktop/winforms/how-to-create-event-handlers-at-run-time-for-windows-forms?view=netdesktop-5.0&branch=pr-en-us-1113
Resource | dotnet-desktop-guide/net/winforms/controls/media/how-to-add-an-event-handler/visual-studio-properties-events-click.png | https://review.docs.microsoft.com/en-us/dotnet/desktop/winforms/controls/media/how-to-add-an-event-handler/visual-studio-properties-events-click.png?view=netdesktop-5.0&branch=pr-en-us-1113
Resource | dotnet-desktop-guide/net/winforms/media/shared/visual-studio-chevron-button.png | https://review.docs.microsoft.com/en-us/dotnet/desktop/winforms/media/shared/visual-studio-chevron-button.png?view=netdesktop-5.0&branch=pr-en-us-1113
Resource | dotnet-desktop-guide/net/winforms/media/shared/visual-studio-ellipsis-button.png | https://review.docs.microsoft.com/en-us/dotnet/desktop/winforms/media/shared/visual-studio-ellipsis-button.png?view=netdesktop-5.0&branch=pr-en-us-1113
Resource | dotnet-desktop-guide/net/winforms/media/shared/visual-studio-events-button.png | https://review.docs.microsoft.com/en-us/dotnet/desktop/winforms/media/shared/visual-studio-events-button.png?view=netdesktop-5.0&branch=pr-en-us-1113
Toc | dotnet-desktop-guide/net/winforms/toc.yml | https://review.docs.microsoft.com/en-us/dotnet/desktop/winforms/toc.json?view=netdesktop-5.0&branch=pr-en-us-1113

### CODE REVIEW

File: dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  - "events [Windows Forms], connecting multiple to single event handler"
+  - "event handlers [Windows Forms], connecting events to"
+  - "menus [Windows Forms], event-handling methods for multiple menu items"
+  - "Windows Forms controls, events"
+  - "menu items [Windows Forms], multicasting event-handling methods"
+---
+
+# How to handle an event (Windows Forms .NET)
+
+Events for controls (and for forms) are generally set through the Visual Studio Visual Designer for Windows Forms. Setting an event through the Visual Designer is known as handling an event at design-time. You can also handle events dynamically in code, known as handling events at run-time.
```
Comment (luisquintanilla) [inline dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md:23]: Do you need the last sentence here since you mention it again in the next paragraph?

File: dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+In addition to handling events using the Windows Forms Designer in Visual Studio, you can also create an event handler at run-time. An event created at run-time allows you to connect event handlers dynamically based on what your app is currently doing.
+
+[!INCLUDE [desktop guide under construction](../../includes/desktop-guide-preview-note.md)]
+
+## Designer
```
Comment (luisquintanilla) [inline dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md:29]: Suggestion: Make heading more descriptive. Below is an idea, though it may be a bit wordy. Feel free to cut / wordsmith to whatever you thing is appropriate

```suggestion
## Handle events in Visual Studio Visual Designer
```

File: dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+01. Open the Visual Designer of the form containing the control to change.
+01. Select the control.
+01. Change the **Properties** toolbox mode to **Events** by pressing the events button (:::image type="icon" source="../media/shared/visual-studio-events-button.png" border="false":::).
+01. Find the event containing the handler you want to remove, for example, the **Click** event:
+
+    :::image type="content" source="media/how-to-add-an-event-handler/visual-studio-properties-events-click.png" alt-text="Visual Studio properties toolbox shown with the events mode enabled and the click event.":::
+
+01. Right-click on the event and choose **Reset**.
+
+## Programmatic
```
Comment (luisquintanilla) [inline dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md:65]: Similar comment to "Designer" H2. Make it more descriptive. i.e. "Handle events programmatically", "Handle events at run-time", "Handle events (code-first)"

File: dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+In Visual Basic, the handler is attached to a control's event in the form's code-behind file, where the event handler code is declared. Multiple `Handles` keywords can be added to the event handler code to use it with multiple events. The Visual Designer will generate the `Handles` keyword for you and add it to the event handler. However, you can easily do this yourself to any control's event and event handler, as long as the signature of the handler method matches the event. For more information about Visual Basic events, see [Events (Visual Basic)](/dotnet/visual-basic/programming-guide/language-features/events/)
+
+The following code is for Visual Basic. If you're viewing this article in C# language mode, you may not see the code below. This code demonstrates how the same method can be used as a handler for two different [`Button.Click`](xref:System.Windows.Forms.Control.Click) events:
+
+:::code language="vb" source="snippets/how-to-add-an-event-handler/vb/Form2.vb" id="MultipleHandlers":::
```
Comment (luisquintanilla) [inline dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md:108]: I'm confused by the purpose of this section. Can we do something like the other snippets which have a C# & VB version? Also, it's showing up when the language I've selected is C#.

File: dotnet-desktop-guide/net/winforms/controls/how-to-add-a-picture-to-a-control.md [hunk 5]
```diff
@@ -21,7 +21,7 @@ Several Windows Forms controls can display images. These images can be icons tha
 
 ## Designer
 
-In the **Properties** window of Visual Studio, select the **Image** or **BackgroundImage** property of the control, and then select the ellipsis (![Ellipsis button in Visual Studio](../media/visual-studio-ellipsis-button.png)) to display the **Select Resource** dialog box and then select the image you want to display.
+In the **Properties** window of Visual Studio, select the **Image** or **BackgroundImage** property of the control, and then select the ellipsis (:::image type="icon" source="../media/shared/visual-studio-ellipsis-button.png" border="false":::) to display the **Select Resource** dialog box and then select the image you want to display.
```
Comment (luisquintanilla) [inline dotnet-desktop-guide/net/winforms/controls/how-to-add-a-picture-to-a-control.md:24]: Can this be formatted similar to the Text displayed article where it's a numbered list?

**Review by luisquintanilla — COMMENTED** [review]
@adegeo looks good to me. Added a few comments.

File: dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md (hunk 4 above)
Comment (adegeo) [reply dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md:108]: I wonder if they changed something with the language switcher. they used to NEVER show up. I wonder if you don't have multiple language entries one-line-after-the-other if it just displays it? Anyhooo, the reason I have this section is because VB technically has two ways of creating handlers, declaring them with `AddHandler` which is the same as C#'s `+=` operator, and the `Handles` keyword which C# has no equivalent of. VB in Visual Studio uses the `Handles` version. 😦

File: dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md (hunk 2 above)
Comment (adegeo) [reply dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md]: How does it sound now?

File: dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md (hunk 3 above)
Comment (adegeo) [reply dotnet-desktop-guide/net/winforms/controls/how-to-add-an-event-handler.md]: @luisquintanilla How does this sound? I used the same format on the other article too. I'm trying not to be too wordy on the headings because they're wrapping.

**Review by luisquintanilla — COMMENTED** [review]
Changes look good. The only item outstanding is VB content showing up for C#.

image

image

**Comment by adegeo** [comment]
@luisquintanilla Really there isn't anything I can do about it. But I have to call it out. I at least added it to a section named **Visual Basic** so it's clearer.

**Review by luisquintanilla — APPROVED** [review]

**Comment by adegeo** [comment]
@luisquintanilla thanks a bunch for the good review!
